src/styles/abstracts/_mixins.scss

```diff
@@ -38,3 +38,244 @@
     text-overflow: ellipsis;
     white-space: nowrap;
 }
+
+@mixin content-padding($line-height) {
+    padding: calc(#{$line-height} * 0.25) 0;
+
+    // Resets defaults
+    margin-block-end: 0;
+    margin-block-start: 0;
+    margin-inline-end: 0;
+    margin-inline-start: 0;
+}
+
+@mixin content-margin($line-height) {
+    margin: calc(#{$line-height} * 0.25) 0;
+}
+
+/** A set of opinionated styles to be used privately by Octuple components */
+@mixin octuple-chrome {
+    .oc-content {
+        padding: 0 1.6rem;
+    }
+
+    .oc-display {
+        font-style: normal;
+        font-weight: $text-font-weight-light;
+        font-size: $text-font-size-11;
```

Comment:
yikes. do we have this many font sizes?
I prefer the way Angular Material handles fonts. Named font size ("header", "caption", things like that) that include size, weight, line-height, etc. Then you call a mixin which will give you the appropriate font rules based on the named font size. Additionally, we could expose those things as css vars for greater customizability

Reply:
Yes. Lets do this in the refinement phase as the SASS vars are private